app/internal/wysiwyg_editor.py

```diff
@@ -0,0 +1,44 @@
+import bleach
+
+
+def clean_html(html: str) -> str:
+    """
+    This function is getting an html string and checking
+    for malicious and not allowed tags, attrs and styles
+    and then returns it cleaned without the not allowed parameters.
+
+    Args:
+        html (str): string from a post request made by the summernote editor.
+
+    Returns:
+        str: bleached string of html.
+    """
+
+    allowed_tags = [
```

Comment:
Can the lists below change at some point?

Reply:
To my honest opinion I thought about it, and it doesn't seem like you would want to change them, have you thought about something to add?

Reply:
I agree, it's seems pretty much **constant** to me as well.